ansible/group_vars/node-db.yml

```diff
@@ -17,9 +17,9 @@ postgres_ha_databases:
 
 postgres_ha_backup: false
 
+
```

Comment:
Why is that?

Reply:
ah probably because of my editor, i can undo this line break